server/protocols/webadmin/webadmin-mailbox/pom.xml

```diff
@@ -156,5 +156,38 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>swagger-json</id>
```

Comment:
please don't use profile, I'm so bored of having to specify profiles

Reply:
Then you are going to do a lot of tasks when you only want to compile your code, and it will make you lose some time.